notebooks/MAST/astroqueryMAST/beginner_search_astroqueryMAST/beginner_search_astroqueryMAST.ipynb

```diff
@@ -0,0 +1,395 @@
+   "source": [
+    "## Table of Contents\n",
+    "\n",
+    "* [Imports](#Imports)\n",
+    "\n",
+    "\n",
+    "* [Three ways to search for MAST observations](#Three-Ways-to-Search-for-MAST-Observations)\n",
+    "    - [By Region](#1.-By-Region)\n",
+    "    - [By Object Name](#2.-By-Object-Name)\n",
+    "    - [By Criteria](#crit)\n",
+    "    \n",
+    "    \n",
+    "* [Getting Associated Data Products](#Getting-Associated-Data-Products)\n",
+    "    - [Performing a Product Query](#Performing-a-Product-Query)\n",
+    "    - [Filtering Data Products](#Filtering-the-Data-Products)\n",
+    "    \n",
+    "    \n",
+    "* [Downloading Products](#Downloading-Products)\n",
+    "\n",
+    "\n",
+    "* [Displaying Data](#Displaying-Data)\n",
+    "\n",
+    "\n",
+    "* [Further Reading](#Further-Reading)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The only import we need for the MAST API is `astroquery.mast`. \n",
+    "\n",
+    "`astropy` and `matplotlib` are for opening and plotting the data, respectively."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astroquery.mast import Observations\n",
+    "\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import SymLogNorm\n",
+    "%matplotlib inline "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Three Ways to Search for MAST Observations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. By Region\n",
+    "You can use `query_region` to find objects based on coordinates and radius.  \n",
+    "The coordinates can be given as a string or `astropy.coordinates` object, and the radius as a string or float object. If no radius is specified, the default is 0.2 degrees."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will give a warning that the coordinates are being interpreted as an ICRS coordinate provided in degrees\n",
+    "obsByRegion = Observations.query_region(\"322.49324 12.16683\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# How many results? Let's look at the first ten.\n",
+    "print(\"Number of results:\",len(obsByRegion))\n",
+    "print(obsByRegion[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. By Object Name\n",
+    "\n",
+    "The `query.object` method allows you to search based on object name and optional radius argument.  \n",
+    "The object name is first resolved to sky coordinates, and then the resolved region is queried as above."
```

Comment:
Let's add an additional detail: the string is resolved by a call to the Simbad and NED archives. For some special catalogs like KIC, EPIC, and TIC, a direct lookup using the catalog at MAST is used.

Reply:
Actually, I recommend additional examples in a text cell after, or even a 2nd set of cells, since it can sometimes be a source of confusion, showing an exmaple of how to search for objects from the Kepler, K2, and TESS catalogs.  The source of confusion for users is to include the specific catalog identifier in front of the star number, so the resolver knows what catalog to use.  Examples to use:

"KIC 011446443" = The host star for exoplanet TrES-2 b.
"EPIC 201912552" = The host star for exoplanet K2-18 b.
"TIC 261136679" = The host star for exoplanet pi Men b.